.github/ISSUE_TEMPLATE.md

```diff
@@ -1,20 +1,70 @@
-**Please provide the following information**
+<!--
+###############################################################################
+  WARNING!
+  IGNORING THE FOLLOWING TEMPLATE WILL RESULT IN THE ISSUE BEING CLOSED
+  AS INCOMPLETE/INVALID
+###############################################################################
+-->
+
+## Checklist
+
+<!--
+################################## IMPORTANT ##################################
+
+As you read and fulfill each of the following requirements below,
+put an "x" (without the quotes) between the square brackets.
+
+When doing so, ensure that no space is left between the square brackets and the "x", like so: [x]
+
+Be honest!
+-->
+
+- [ ] I've read the **issue reporting guidelines** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good report with the required information
+- [ ] I've verified that **the issue is not fixed and is reproducible** in the **latest version, 4.2.3**
+- [ ] I've **searched** the issue tracker for similar bug reports including closed ones **to avoid posting a duplicate issue report**
+- [ ] This is **not a support issue or question**, or the [wiki](https://github.com/qbittorrent/qBittorrent/wiki) didn't contain the required information and the [forum](https://qbforums.shiki.hu/) and [subreddit](https://www.reddit.com/r/qBittorrent/) were not able to assist me, so I'm asking here
+- [ ] I have **pasted/attached the settings file and relevant log(s)** (at least the most recent one) in the **Attachments** section at the bottom of the report and filled in all the other required information
```

Comment:
Should the location of the settings file be mentioned if we're requiring it now?

Reply:
That information is already present in the Wiki FAQ, which we assume the user has read before posting.